wcivf/apps/elections/views/postcode_view.py

```diff
@@ -87,6 +88,8 @@ def get(self, request, *args, **kwargs):
                 post_election.election.name, post_election.post.label
             )
 
+            # event.add("dtstamp", datetime.isocalendar())
+            event.add("dtstamp", datetime.now().year)
             event.add("dtstart", post_election.election.start_time)
```

Comment:
I am unclear how these params related to django params for "X-WR-TIMEZONE" which we have already defined. Perhaps DTSTART and DTEND need to concatenate TZID or "X-WR-TIMEZONE" to satisfy the TZID errors. 